src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj

```diff
@@ -24,6 +24,8 @@
     <NoDocumentationFile>true</NoDocumentationFile>
     <UseCommonOutputDirectory>true</UseCommonOutputDirectory>
     <ProjectSystemLayer>VisualStudio</ProjectSystemLayer>
+    <NeedsSigningToRelease>true</NeedsSigningToRelease>
+    <ProducingSignedVsix>true</ProducingSignedVsix>
```

Comment:
Nonshipping is used throughout Roslyn and VS to indicate this, you're inventing a new mechanism - the test projects, etc are already marked with it.